### PR TITLE
Add output states in heartbeat data

### PR DESCRIPTION
**Pull request by khenderick**

### CODE REVIEW

**Comment by khenderick** [comment]
@openmotics-ci test gateway=bugfix/output-state-in-heartbeat frontend=develop

File: src/gateway/webservice.py [hunk 1]
```diff
@@ -2150,7 +2150,7 @@ def get_version(self):
         :rtype: dict
         """
         return {'version': self._gateway_api.get_main_version(),
-                'gateway': '2.10.1'}
```
Comment (svanscho) [inline src/gateway/webservice.py]: Can we put all version number strings in one place? Easier to manage version bumps in the future.

Comment (khenderick) [reply src/gateway/webservice.py]: I agree that eventually we should move this version number to some file that can be generated by e.g. Jenkins and that can be shipped with the code. But I would prefer to do such a change in a separate PR.

Comment (svanscho) [reply src/gateway/webservice.py]: Agreed.

**Pull request merged**